scripts/setup.py

```diff
@@ -414,4 +426,9 @@ def validateParameters(project):
     ios.run()
     flutter = Flutter(project)
     flutter.run()
-    print("=> 🚀 Done! App is ready to be tested 🙌")
+    print('🤖 Generating project...')
+    # Remove the `.github` folder to avoid redundant workflow
+    clean_up('.github')
+    move_project_to_root(project)
+    clean_up('.template')
```

Comment:
how about those files as well? Even some of them will be overwritten by the main project content but would it be better to ensure everything should be cleaned up before 🤓  Any files we can not handle right now could mark with FIXME comment for later 👌 

For example, the LICENSE file will be remained after initializing 🤔:

![image](https://user-images.githubusercontent.com/16315358/163550006-d54d68c3-cf83-47bf-9526-187821735ddb.png)

Reply:
@luongvo updated 🚀